src/feedback/models/base.py

```diff
@@ -316,6 +324,18 @@ class Veranstaltung(models.Model):
     freiefrage2 = models.TextField(verbose_name='2. Freie Frage', blank=True)
     kleingruppen = models.TextField(verbose_name='Kleingruppen', blank=True)
 
+    def get_next_state(self):
```

Comment:
Just use a dict for it where the source state is the key and the targed state is the value.
Whats about the case that there are more then one possible targed state? So you might better return a tuple containing all targed states then a single value.